pages/Company-Info-Page.tsx

```diff
@@ -0,0 +1,70 @@
+import Link from "next/link";
+import Image from "next/image";
+import type { NextPage } from "next";
+import { useState } from "react";
+
+
+
+const Company:NextPage = () => {
+
+    const [Count,setCount] = useState ('هنوز هیچ فایلی انتخاب نشده');
```

Comment:
Take a look at the translations used in `login`, `welcome` or `phoneVerification` pages and add your own translations. Read the code to learn the translation workflow.